lib/basedriver/commands/find.js

```diff
@@ -273,29 +272,28 @@ helpers.findByImage = async function findByImage (b64Template, {
     }
   }
 
-  if (!rect) {
+  if (_.isEmpty(results)) {
     if (multiple) {
       return [];
     }
     throw new errors.NoSuchElementError();
   }
 
-  log.info(`Image template matched: ${JSON.stringify(rect)}`);
-  if (b64Matched) {
-    log.info(`Matched base64 data: ${b64Matched.substring(0, 200)}...`);
-  }
-  const imgEl = new ImageElement(b64Template, rect, score, b64Matched);
+  const elements = results.map((result) => {
```

Comment:
you could destruct result right here, for example:

`results.map((result) => {` -> `results.map(({rect, score, visualization}) => {`

Reply:
😲

🙏 Thank you for teaching me new things